relay/nakama/main.go

```diff
@@ -332,6 +343,17 @@ func initCardinalEndpoints(logger runtime.Logger, initializer runtime.Initialize
 				if err != nil {
 					return logError(logger, "can't read body: %w", err)
 				}
+				var asTx txResponse
+
+				if err = json.Unmarshal(bodyStr, &asTx); err != nil {
+					return logError(logger, "can't decode body as tx response: %w", err)
+				}
+				userID, err := getUserID(ctx)
```

Comment:
I suppose it probably doesn't make sense to extend the functionality of this PR to support this, but I can imagine a use cases where a client may want to specify several user ids that should be sent a "notification" to. Take for example almost any board game like Chess, if `Player1` makes a move you probably want to register `Player2` to also get a notification for the result of that move just because that would increase the liveness of the game at minimal cost.

That being said, letting a client specify which user id's to reply is a feature that seems kind of across both Cardinal and Nakama to me at this point, unclear where it should be. 

If we are going to ignore this for now, should we backlog a ticket to remember to come back to this?

Reply:
I added a ticket: https://linear.app/arguslabs/issue/WORLD-424/decide-which-users-should-see-which-transaction-receipts